types/node-siege/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for node-siege 0.0
```

Comment:
Isn't this at version 0.2 at this point?

Reply:
oh, so there are two different packages with `siege` monicker (it tricked me also),
This one is also legit (node-siege) and bears `0.0.4` version:
https://github.com/npr/node-siege/blob/master/package.json#L4
It's kind of ol', but not deprecated, thx!